src/main/kotlin/uk/gov/dluhc/notificationsapi/rest/SentCommunicationsController.kt

```diff
@@ -41,4 +46,24 @@ class SentCommunicationsController(
         }.let {
             CommunicationsHistoryResponse(communications = it)
         }
+
+    @GetMapping("{communicationId}/")
+    @PreAuthorize(HAS_APPLICATION_SPECIFIC_ERO_ADMIN_AUTHORITY)
+    fun getSentCommunication(
+        @PathVariable eroId: String,
+        @PathVariable communicationId: String,
+        @RequestParam(required = false, defaultValue = "voter-card") sourceType: SourceTypeApi,
```

Comment:
feel like I've seen this before, but what's the reason for the default value?

Reply:
I suspect it's VAC existed before the other source types and we wanted to make things backwards compatible by allowing VAC to continue to call the endpoints without specifying which type of service it was. Not necessary for this endpoint since nothing calls it yet so it doesn't need to be backwards compatible so I'll remove the default.